src/plugins/ui_stateful/clipboard.ts

```diff
@@ -109,6 +109,40 @@ export class ClipboardPlugin extends UIPlugin {
           this.status = "invisible";
         }
         break;
+      case "REPLICATE_CELLS_ABOVE":
+        if (this.getters.getSelectedZones().length === 1) {
+          const zone = this.getters.getSelectedZone();
+          const singleRowSelected = zone.top !== zone.bottom;
+          const copyTarget = {
+            ...zone,
+            bottom: singleRowSelected ? zone.top : zone.top - 1,
+            top: singleRowSelected ? zone.top : zone.top - 1,
+          };
+          this.state = this.getClipboardState([copyTarget], "COPY");
```

Comment:
```suggestion
          const state = this.getClipboardState([copyTarget], "COPY");
```
don't change the clipbaord state

Reply:
Applied the change.
Clipboard state won't be affected now :+1: